src/plugins/home/server/services/tutorials/lib/tutorial_schema.ts

```diff
@@ -124,7 +124,7 @@ export const tutorialSchema = {
   moduleName: Joi.string(),
   isBeta: Joi.boolean().default(false),
   shortDescription: Joi.string().required(),
-  euiIconType: Joi.string(), // EUI icon type string, one of https://elastic.github.io/eui/#/icons
+  euiIconType: Joi.string(), // OUI icon type string, one of https://oui.opensearch.org/#/display/icons
```

Comment:
Shouldn't we change the name of this key from `euiIconType` to `ouiIconType`? Not sure about the repercussions it could have this change.

Reply:
No, not yet. We're still sticking with the EUI namespace until we have a firm date for a 3.0 release, because switching the namepaces will make backporting much more difficult.